terraform/main.tf

```diff
@@ -86,6 +86,8 @@ locals {
       osdisk_size_controller  = "150"
       num_builders_x86        = 1
       num_builders_aarch64    = 1
+      binary_cache_public_key = "foo:changeme"
```

Comment:
We could set this to "ghaf-infra-dev:EdgcUJsErufZitluMOYmoJDMQE+HFyveI/D270Cr84I=" too for now

Reply:
Ok, will do

Reply:
Done, and added a TODO to set this to the real pubkeys used there (once we have them)